src/components/Charts/Keyboard.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div
+    :id="id"
+    :class="className"
+    :style="{height: height, width: width}"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import echarts, { ECharts, EChartOption } from 'echarts'
+import { ResizeMixin } from './mixins/resize'
+
+@Component({
+  mixins: [ResizeMixin]
+})
+export default class Keyboard extends Vue {
```

Comment:
Prefer use `extends mixins(ResizeMixin)`, in this way, the variables and methods from the mixin can be access from this parent class, and you don't need to define chart again.  See `layout/index.vue` for an example (which can access compute property `sidebar` from mixin)